integration/integration.go

```diff
@@ -293,24 +293,28 @@ func (i *Infrastructure) initNWO() {
 		switch label {
 		case "fsc":
 			// treat fsc as special
-			fscTopology = topology
-			continue
+			factory, ok := i.PlatformFactories["fsc"]
+			Expect(ok).To(BeTrue(), "expected to find platform [%s]", label)
```

Comment:
do we need this assertion? I think it should work without FSC factories (though I don't remember whether we have any such tests)

Reply:
true - a few lines before we register fsc factory in case it does not exist